### PR TITLE
WIP [BEAM-6056] Rename v1_13_1 to v1p13p1.

### PR DESCRIPTION
**Pull request by aaltay**
**Please** add a meaningful description for your change here

------------------------

Follow this checklist to help us incorporate your contribution quickly and easily:

 - [ ] Format the pull request title like `[BEAM-XXX] Fixes bug in ApproximateQuantiles`, where you replace `BEAM-XXX` with the appropriate JIRA issue, if applicable. This will automatically link the pull request to the issue.
 - [ ] If this contribution is large, please file an Apache [Individual Contributor License Agreement](https://www.apache.org/licenses/icla.pdf).

It will help us expedite review of your Pull Request if you tag someone (e.g. `@username`) to look at it.

Post-Commit Tests Status (on master branch)
------------------------------------------------------------------------------------------------

Lang | SDK | Apex | Dataflow | Flink | Gearpump | Samza | Spark
--- | --- | --- | --- | --- | --- | --- | ---
Go | [Build Status](https://builds.apache.org/job/beam_PostCommit_Go_GradleBuild/lastCompletedBuild/) | --- | --- | --- | --- | --- | ---
Java | [Build Status](https://builds.apache.org/job/beam_PostCommit_Java_GradleBuild/lastCompletedBuild/) | [Build Status](https://builds.apache.org/job/beam_PostCommit_Java_ValidatesRunner_Apex_Gradle/lastCompletedBuild/) | [Build Status](https://builds.apache.org/job/beam_PostCommit_Java_ValidatesRunner_Dataflow_Gradle/lastCompletedBuild/) | [Build Status](https://builds.apache.org/job/beam_PostCommit_Java_ValidatesRunner_Flink_Gradle/lastCompletedBuild/) [Build Status](https://builds.apache.org/job/beam_PostCommit_Java_PVR_Flink/lastCompletedBuild/) | [Build Status](https://builds.apache.org/job/beam_PostCommit_Java_ValidatesRunner_Gearpump_Gradle/lastCompletedBuild/) | [Build Status](https://builds.apache.org/job/beam_PostCommit_Java_ValidatesRunner_Samza_Gradle/lastCompletedBuild/) | [Build Status](https://builds.apache.org/job/beam_PostCommit_Java_ValidatesRunner_Spark_Gradle/lastCompletedBuild/)
Python | [Build Status](https://builds.apache.org/job/beam_PostCommit_Python_Verify/lastCompletedBuild/) | --- | [Build Status](https://builds.apache.org/job/beam_PostCommit_Py_VR_Dataflow/lastCompletedBuild/) </br> [Build Status](https://builds.apache.org/job/beam_PostCommit_Py_ValCont/lastCompletedBuild/) | [Build Status](https://builds.apache.org/job/beam_PostCommit_Python_VR_Flink/lastCompletedBuild/) | --- | --- | ---

### CODE REVIEW

**Comment by aaltay** [comment]
Run Java PreCommit

**Comment by swegner** [comment]
Run Java PreCommit

**Comment by swegner** [comment]
I was having trouble repro'ing this locally. Just running the `:beam-sdks-java-core:compileJava` target directly seems to pass, but running the full `:javaPreCommit` triggers the failure.

* `:beam-sdks-java-core:compileJava` (succeeds): https://gradle.com/s/2itp5l5zlftie
* `:javaPreCommit` (fails): https://gradle.com/s/gfnbbedx3oru6

My hypothesis is that some additional task included in the graph must have some side-effect; I'm going to continue to try and narrow down the differences between builds.

**Comment by kennknowles** [comment]
I built a branch off this and rebased (resolving the conflict) and did not repro the failure on Jenkins.

File: buildSrc/src/main/groovy/org/apache/beam/gradle/GrpcVendoring.groovy [hunk 1]
```diff
@@ -72,7 +72,7 @@ class GrpcVendoring {
     // those libraries may provide. The 'validateShadedJarDoesntLeakNonOrgApacheBeamClasses'
     // ensures that there are no classes outside of the 'org.apache.beam' namespace.
 
-    String prefix = "org.apache.beam.vendor.grpc.v1_13_1";
+    String prefix = "org.apache.beam.vendor.grpc.v1p13p1";
```
Comment (swegner) [inline buildSrc/src/main/groovy/org/apache/beam/gradle/GrpcVendoring.groovy]: It seems this prefix isn't currently used.. that looks suspicious

**Comment by swegner** [comment]
It looks like the `jar` task and `shadowJar` task on `:beam-vendor-grpc-1_13_1` both have the same `archivePath` (vendor/grpc-1_13_1/build/libs/beam-vendor-grpc-1_13_1-0.1.jar); so I believe the non-shaded JAR is overwriting the shaded JAR and causing issues.

So, new hypothesis is that the bug is one of..

- The tasks need to have unique archivePath's
- Tasks should only depend on the vendored shadowJar and the jar task should never execute

My next steps are to compare the behavior on master.

File: buildSrc/src/main/groovy/org/apache/beam/gradle/GrpcVendoring.groovy (hunk 1 above)
Comment (swegner) [reply buildSrc/src/main/groovy/org/apache/beam/gradle/GrpcVendoring.groovy]: Nevermind, this same config is embedded below. I don't think this is an issue.

**Comment by swegner** [comment]
Latest status: it appears that `:beam-vendor-grpc-1_13_1:assemble` depends on `:beam-vendor-grpc.1_13_1:jar` for some reason and that is triggering the execution within `:javaPreCommit`.

I believe having `:beam-vendor-grpc.1_13_1:jar` in the execution graph is an indication of a bug; I'll continue investigating.

**Comment by aaltay** [comment]
Closing this in favor of https://github.com/apache/beam/pull/7324

**Pull request closed without merging**